experimental/FTheoryTools/src/FamilyOfG4Fluxes/properties.jl

```diff
@@ -19,24 +26,81 @@ A family of G4 fluxes:
   - Elementary quantization checks: satisfied
   - Verticality checks: failed
   - Non-abelian gauge group: broken
+  - Tadpole constraint: not analyzed
 
 julia> is_well_quantized(gf)
 true
+
+julia> m1 = matrix_integral(gf);
+
+julia> m2 = matrix_rational(gf);
+
+julia> gf2 = family_of_g4_fluxes(qsm_model, m1, m2, check = false)
+A family of G4 fluxes:
+  - Elementary quantization checks: not executed
+  - Verticality checks: not executed
+  - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole constraint: not analyzed
+
+julia> is_well_quantized(gf2, check = false)
+true
 ```
 """
-function is_well_quantized(fgs::FamilyOfG4Fluxes)
-  @req has_attribute(fgs, :is_well_quantized) "Cannot (yet) tell if this family of G4-fluxes is well-quantized"
-  return get_attribute(fgs, :is_well_quantized)
+function is_well_quantized(fgs::FamilyOfG4Fluxes; check::Bool = true)
```

Comment:
```suggestion
@attr Bool function is_well_quantized(fgs::FamilyOfG4Fluxes; check::Bool = true)
```
This can make use of the new `@attr` functionality. I.e. add this and remove all other attribute handling with `:is_well_quantized` from the  function body